sympy/simplify/simplify.py

```diff
@@ -525,7 +525,10 @@ def simplify(expr, ratio=1.7, measure=count_ops, rational=False):
     from sympy import Sum, Product
 
     if not isinstance(expr, Basic) or not expr.args:  # XXX: temporary hack
-        return expr
+        if expr == False or expr == True:
```

Comment:
This will raise AttributeError each time `expr` is 0 or 1. There should be no error in those cases.